repository/git.go

```diff
@@ -207,6 +207,34 @@ func (repo *GitRepo) RmConfigs(keyPrefix string) error {
 	return err
 }
 
+//ListRemotes lists all the local remotes
+func (repo *GitRepo) ListRemotes() (map[string]string, error) {
+	out, err := repo.runGitCommand("remote", "--verbose")
+
+	if err != nil {
+		return nil, err
+	}
+
+	splitRemotes := strings.Split(out, "\n")
+
+	remotesMap := make(map[string]string, len(splitRemotes))
+
+	for _, line := range splitRemotes {
+
+		remoteValues := strings.Fields(line)
+		if len(remoteValues) != 3 {
+			return nil, fmt.Errorf("Error in remote: %s", line)
```

Comment:
`"Error in remote: %s"` doesn't actually said what the problem is. We already know that there is an error because we get an error. Also, error message in go should not start with a capital letter.

You can say `"failed to parse remote %s"`

Reply:
Yeah I think you are right we need to change the error message.